src/taxonomies/taxonomiesService.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash/fp';
+
+import * as terrasoApi from 'terrasoBackend/api';
+
+import { extractTerms } from './taxonomiesUtils';
+
+export const fetchTermsForTypes = ({ types }) => {
+  const query = `
+    query taxonomyTerms($types: [String]!){
+      taxonomyTerms(type_In: $types) {
+        edges {
+          node {
+            slug
+            type
+            valueOriginal
+            valueEn
+            valueEs
```

Comment:
question: can we make this a fragment that we can reuse / abstract the language list?

Reply:
Yes we can